### ISSUE TITLE
Using Wikipedia more to highlight request on Alaveteli sites

### ISSUE THREAD

**Issue by Gemmamysoc** [mysociety/alaveteli-experiments#99]
## Q: [What question are you trying to understand?]

[Explain how you will measure this]

[What do we need to do to be able to take the measurements?]

**Comment by garethrees** [comment]
This looks like a dup of https://github.com/mysociety/alaveteli-experiments/issues/99

**Comment by Gemmamysoc** [comment]
Just for info/background, Wikipedia brings the following % of traffic (aka Sessions) to these Alaveteli sites:

RightToKnow: 0.07%
WhatDoTheyKnow: 0.03% (only based on 0.84% of sessions)
KiMiTud: 0.01%
IPZ (Croatia): 0.00%
AskYourGov: 0.00%

Could we work to increase these percentages?!

**Comment by andreaspavlou** [comment]
This could maybe be done with these guys who have been actively adding links to wikipedia pages (links of FOIA requests could work too!) : http://www.balcanicaucaso.org/eng/Areas/Serbia/Wiki4MediaFreedom-in-10-points-175569

**Comment by Gemmamysoc** [comment]
Ooh, great! Are you in touch with the organisers of this?

**Comment by andreaspavlou** [comment]
Can do, will email now. 



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by henare** [comment]
For comparison: On https://theyvoteforyou.org.au/ an amazing volunteer came along one day and put links on MP Wikipedia pages back to their profile on They Vote For You. In the last month 153 sessions out of 3,052 were from Wikipedia (5%).

>Just for info/background, Wikipedia brings the following % of traffic (aka Sessions) to these Alaveteli sites:
>
>RightToKnow: 0.69% (based on 85.41% of sessions)

@Gemmamysoc I don't really understand these statistics. What time period do we mean? Is this 0.69% of all traffic? Is it based on 85.41% of referral sessions?

**Comment by Gemmamysoc** [comment]
Hey @henare. I looked at Referral traffic on RightToKnow's Google Analytics from 1st October 2012 until now. It only displayed 84.41% sessions to me yesterday, but I'm looking again now and it seems to be saying 0.25% of traffic has come from Wikipedia and is basing that on 100% of sessions. Google Analytics can be so confusing sometimes, grr!

**Comment by henare** [comment]
>Google Analytics can be so confusing sometimes, grr! 

Sure can! It's show as 0.07% for that period for me:

![selection_001](https://cloud.githubusercontent.com/assets/48945/20416536/928f2436-ad93-11e6-9f12-d902608c68b3.png)

In any case, less than 1% has definite room for improvement. Especially since we see 5% on another one of our projects.

Of course the important thing is to not be spammy. The They Vote For You links are clearly helpful so I think that's why they're still there and also bringing in visitors.

**Comment by Gemmamysoc** [comment]
Yes, very important not to be spammy. Links to request should only be added if they truly are relevant. Wikipedia will soon notice if the links aren't relevant anyway!

**Comment by Gemmamysoc** [comment]
After speaking with OBC Transeuropa, they suggested that people running Alaveteli sites should contact their local [Wikimedia chapters](https://meta.wikimedia.org/wiki/Wikimedia_chapters), to inform them of the resources/responses available on their site. Wikimedia could then let their Wikipedians know about this source of material that they could use when editing Wikipedia. Getting involved in the [Wikipedian in Residence](https://outreach.wikimedia.org/wiki/Wikipedian_in_Residence) programme might be another avenue. 

OBC Transeuropa might run a Wiki edit-a-thon on Access to Information in London next year, which could use Alaveteli site content as source material.
